lib/server.js

```diff
@@ -336,13 +334,6 @@ Server.prototype.start = function start(cb) {
       self._baseHttpServer.listen(self._listenPort, function(err) {
         if (err) return callback(err);
 
-        var address = this.address();
-        console.log('%s: StrongLoop PM v%s (API v%s) listening on port `%s`',
```

Comment:
:+1:   I thought I made this change already, but it must have been executor or central..